tests/fixtures/saas/zendesk_fixtures.py

```diff
@@ -88,3 +100,66 @@ def zendesk_dataset_config(
     )
     yield dataset
     dataset.delete(db=db)
+
+
+@pytest.fixture(scope="function")
+def zendesk_create_erasure_data(
+    zendesk_connection_config: ConnectionConfig, zendesk_erasure_identity_email: str
+) -> None:
+
+    zendesk_secrets = zendesk_connection_config.secrets
+    base_url = f"https://{zendesk_secrets['domain']}"
+    headers = {
+        "Content-Type": "application/json",
+        "Authorization": "Basic {}".format(
```

Comment:
The `requests` library has a shorthand for this
https://requests.readthedocs.io/en/latest/user/authentication/

you should just able to define the auth then use it in your request
```
auth = zendesk_secrets['username'], zendesk_secrets['api_key']
...
requests.post(url=..., auth=auth, json=...)
```